dist/angular-breadcrumb.js

```diff
@@ -212,6 +216,7 @@ function BreadcrumbDirective($interpolate, $breadcrumb, $rootScope) {
                     scope.steps = $breadcrumb.getStatesChain();
                     angular.forEach(scope.steps, function (step) {
                         if (step.ncyBreadcrumb && step.ncyBreadcrumb.label) {
+                            viewScope.ncyStateParams = $breadcrumb.getStateParams();
```

Comment:
Hi,
I use translations in the breadcrumb label, eg. `label: {{ 'breadcrumb.login.admin' | translate }}` by changing the last part of my translation key from stateParams (`admin/company/user`). I had some troubles when I try to inject the stateParams from controller scope and I was getting `undefined` in some cases.